__tests__/app.test.js

```diff
@@ -430,4 +430,36 @@ describe("app", () => {
         });
     });
   });
+  describe("DELETE /api/comments/:comment_id", () => {
+    test("Status 204 - Deletes a given comment", () => {
+      return request(app)
+        .delete("/api/comments/1")
+        .expect(204)
+        .then(() => {
+          return connection.query(
+            "SELECT * FROM comments WHERE comment_id = $1",
+            [1]
```

Comment:
I like this extra assertion to make sure the comment is actually gone!
Not that I want to get you out of the habit - There isn't a true need for a parameterised query here as you are in total control and no risk of SQL Injection - not a problem though!